src/app/static/src/app/store/modelRun/mutations.ts

```diff
@@ -68,6 +73,10 @@ export const mutations: MutationTree<ModelRunState> = {
         stopPolling(state);
         Object.assign(state, initialModelRunState());
         state.ready = true;
+    },
+
+    [ModelRunMutation.ClearResult](state: ModelRunState) {
+        state.result = null;
```

Comment:
I'm surprised this is the only state that needs to be reset here.

Reply:
Should maybe do the plotting selections too, is that what you mean? Possible that filter options selected for previous run won't be relevant any more. Not sure we've been resetting those anyway - will have a look.

Reply:
I think we're good with plotting selections actually - there's some code to update for the barchart specifically, and I think the Choropleth and Bubble chart will work around selected filter options which are no longer valid.

Reply:
OK, thanks for checking. I wasn't sure if other state that got updated by the modelrun would need to be reset also or if that was occurring automatically downstream (if that makes sense).

Reply:
Yeah, you're right, the ResetOutputs mutation in root gets triggered by this mutation, which wipes everything upstream.